package.json

```diff
@@ -16,8 +16,9 @@
     "postinstall": "patch-package",
     "pre-commit": "yarn lint",
     "start": "react-native start",
-    "test": "jest connectors/**/* src/**/* scripts/* --detectOpenHandles --forceExit",
+    "test": "jest connectors/**/* src/**/* scripts/* --detectOpenHandles --forceExit --coverage",
```

Comment:
adding `--coverage` by default would slow down local tests in `--watch` mode, why do we need to add it here compared to putting it in the action's code?

Reply:
true, I guess we don't need it in the CI either anyway since we don't take any action with the result